WebContent/WEB-INF/jsp/excelImport.jsp

```diff
@@ -277,6 +278,16 @@
 			}
 		 });
 	});
+
+    $( document ).ready(function() {
```

Comment:
@mcritchlow Yes. It'll validate the form for date validation and missing fields (https://github.com/ucsdlib/damsmanager/blob/d8f7f276d768a197bad40554d102218e45f47cd7/WebContent/WEB-INF/jsp/excelImport.jsp#L21-L150). And once ingest is finished, the progress report form will be hidden and the ingest form will be shown up again, which need to re-enable the submit form button.